pulpcore/tests/unit/serializers/test_base.py

```diff
@@ -108,3 +115,27 @@ def test_ignored_fields_no_side_effects():
     initial_data = {"field1": 1, "csrfmiddlewaretoken": 2}
     defined_fields = {"field1": 1}
     validate_unknown_fields(initial_data, defined_fields)
+
+
+class TestGetOrCreateSerializerMixin(TestCase):
```

Comment:
Can you try if this creates valid pytest tests?
```suggestion
class TestGetOrCreateSerializerMixin():
```

Reply:
Looks like the setUp function may not be called with that name.
Also pytest really suggests to move to fixtures.

Reply:
I think this doc applies here.
https://pytest-django.readthedocs.io/en/latest/helpers.html#db

Reply:
Thanks, rewrote it the fixture way.